src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
```

Comment:
>  This way, if the user scratches on beat, the scratch would be heard on beat.

How can the user scratch on beat without listening to the results on headphone?
We can assume that the user listens to the master mix on headphone expects that if he does not hear double beats, the crowd on the does not hear it as well. 

> Closed back headphones do not prevent feeling bass frequencies from loud subwoofers. 

Yes, but if you hear a clear kick base in your closed headphones and the outside is damped by a reasonable amount, you schould be able to distinguish it. That is the reason for using closed headphones.

> I'd imagine that in most cases the difference between output latencies between sound cards running at the same sample rate and buffer size would be very small.

In general this is correct, but not if you try Bluetooth headphones. It is not an important use case, but Mixxx 2.0 support it.